src/mozmlops/templates/Dockerfile

```diff
@@ -4,12 +4,17 @@
 # If you need more help, visit the Dockerfile reference guide at
 # https://docs.docker.com/go/dockerfile-reference/
 
+# This line specifies your base image.
+# For outerbounds, you should specify the platform as shown, even though that is not recommended online,
+# or else flows run into issues (especially when images are built on Macs and then run on Outerbounds's remote node pool machines).
 FROM --platform=linux/amd64 python:3.10.11-bullseye
 
 # Keeps Python from buffering stdout and stderr to avoid situations where
 # the application crashes without emitting any logs due to buffering.
 ENV PYTHONUNBUFFERED=1
 
+# Puts the dependencies from these two requirements lists into the Docker environment for installation
 COPY requirements.outerbounds.txt requirements.outerbounds.txt
 
+# Installs aforementioned dependencies
 RUN python -m pip install -r requirements.txt 
```

Comment:
I just realized that this command should be:
`RUN python -m pip install -r requirements.outerbounds.txt`